runtimes/pythonrt/py-sdk/autokitteh/hubspot.py

```diff
@@ -0,0 +1,35 @@
+"""Initialize a Hubspot client, based on an AutoKitteh connection."""
+
+import os
+
+from hubspot import HubSpot
+
+from .connections import check_connection_name, refresh_oauth
+from .errors import ConnectionInitError, OAuthRefreshError
+
+
+def hubspot_client(connection: str, **kwargs) -> HubSpot:
+    """Initialize a Hubspot client, based on an AutoKitteh connection.
+
+    Args:
+        connection: AutoKitteh connection name.
+
+    Returns:
+        Hubspot SDK client.
+
+    Raises:
+        ValueError: AutoKitteh connection name is invalid.
+        ConnectionInitError: AutoKitteh connection was not initialized yet.
+    """
+    check_connection_name(connection)
+
+    access_token = os.getenv(connection + "__oauth_AccessToken")
```

Comment:
This `access_token` isn't actually being used. Instead, you should check if the refresh token exists (`connection + "__oauth_RefreshToken"`).

Reply:
Done